src/constants.js

```diff
@@ -160,6 +161,7 @@ export const EmoteMenuTips = {
 
 export const SettingDefaultValues = {
   [SettingIds.ANON_CHAT]: false,
+  [SettingIds.AUTO_JOIN_RAIDS]: true,
```

Comment:
we should not perform automatic actions by default

Reply:
Isn't the default Twitch behavior to automatically join you to a raid? If so, should we then change this back to be an "Auto Leave" setting with the default to off? Seems like most if not all BTTV settings represent default Twitch behavior when set to off.

Reply:
you're right. i read this as inverted. this looks correct

Reply:
@night Just to clarify, you believe it is correct to leave this as "Auto Join Raids" with a default of true, instead of "Auto Leave Raids" with a default of false?